tensorflow-core/tensorflow-core-api/src/main/java/org/tensorflow/op/core/Constant.java

```diff
@@ -1356,7 +1356,7 @@
   }
 
   private Constant(Operation operation) {
-    super(operation);
+    super(operation, OP_NAME);
```

Comment:
This change seems a little out of place. Is it a bug fix?

Reply:
It's to match the generation (which hasn't been committed yet).  The reason it's there is that `RawOp`'s constructor checks the passed name against the actual name of the op, since it's theoretically possible for those not to match if you make new op classes or something weird.